recipes/rtmidi/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  "4.0.0":
+    url: "http://www.music.mcgill.ca/~gary/rtmidi/release/rtmidi-4.0.0.tar.gz"
+    sha256: "370cfe710f43fbeba8d2b8c8bc310f314338c519c2cf2865e2d2737b251526cd"
+patches:
+  "4.0.0":
+    - patch_file: "patches/0b5d67c.patch"
```

Comment:
Can you please give the patch files a more descriptive name?
Also, please prepend them with the version as it looks like most/all of them won't be needed anymore for the next version.